### PR TITLE
Added Gulp with minify script and moved js to dist folder. Exposed 'w…

### PR DESCRIPTION
**Pull request by 200even**
…indow.eventListenerOptionsSupported' method. Corrected spelling of 'supported'.

### CODE REVIEW

File: dist/default-passive-events.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 (function() {
+  window.eventListenerOptionsSupported = eventListenerOptionsSupported;
```
Comment (zzarcon) [inline dist/default-passive-events.js]: I think this is out of scope from the library and also don't like the fact of exporting the function to the global scope

File: Gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var gulp = require("gulp");
+var minifyJs = require("gulp-minify");
```
Comment (zzarcon) [inline Gulpfile.js]: Use `const` instead

File: dist/default-passive-events.js (hunk 1 above)
Comment (200even) [inline dist/default-passive-events.js]: Agreed. This was helpful for my specific purpose but is outside the scope.

File: dist/default-passive-events.js [hunk 3]
```diff
@@ -13,6 +11,10 @@
       passive: true,
       capture: false
     };
+    var methodAsString = '' + superMethod;
+    console.log("Method: " + methodAsString);    
+    if(methodAsString.indexOf("preventDefault") !== -1){
+    }
```
Comment (zzarcon) [inline dist/default-passive-events.js]: What is all this for? :)

File: dist/default-passive-events.js [hunk 4]
```diff
@@ -23,20 +25,20 @@
       options.capture = useCapture !== undefined ? useCapture : defaultOptions.capture;
       
       superMethod.call(this, type, listener, options);
-    };
+      };
   }
 
   function eventListenerOptionsSupported() {
-    var sopported = false;
+    var supported = false;
```
Comment (zzarcon) [inline dist/default-passive-events.js]: Ups, thanks!

File: Gulpfile.js [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+const MINIFY_JS = require("gulp-minify");
 
-gulp.task('minify-js', function(){
-    return gulp.src('dist/default-passive-events.js')
-        .pipe(minifyJs())
-        .pipe(gulp.dest('dist'));
+GULP.task('minify-js', function(){
+    return GULP.src('dist/default-passive-events.js')
+        .pipe(MINIFY_JS())
+        .pipe(GULP.dest('dist'));
```
Comment (zzarcon) [inline Gulpfile.js]: Thanks for changing the `var` to `const` but the names should stay in lowercase. Sorry for not letting you beforehand

Comment (200even) [inline Gulpfile.js]: I used uppercase for the `const` names based on the [commonly accepted convention](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const).

    // NOTE: Constants can be declared with uppercase or lowercase, but a common
    // convention is to use all-uppercase letters.

File: dist/default-passive-events.js (hunk 3 above)
Comment (200even) [inline dist/default-passive-events.js]: Oops!

File: Gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const GULP = require("gulp");
+const MINIFY_JS = require("gulp-minify");
+
+GULP.task('minify-js', function(){
+    return GULP.src('dist/default-passive-events.js')
+        .pipe(MINIFY_JS())
+        .pipe(GULP.dest('dist'));
+});
```
Comment (zzarcon) [inline Gulpfile.js]: Fine for me 👍
